### DISCUSSION TITLE
How to find literature references (or longer descriptions) for GO terms?

### DISCUSSION THREAD

**Discussion by jasperhyp** [geneontology/helpdesk#434] [Q&A]
Hi!  I am trying to find some literature references and longer descriptions of GO terms to make machines understand them better.  For instance, if a GO terms has `[PMID:XXX]` in its definition, I can just go to the article and search for it.  However, a lot of GO terms are only annotated with "ISBN" or "GOC" (the ones annotated with e.g. EC, RHEA, TC are easier to deal with because I can go to those databases to find PMIDs).  Any suggestion would be greatly appreciated!

**Comment by suzialeksander** [comment]
Hi @jasperhyp 

Have you tried using the ontology files: http://current.geneontology.org/ontology? These ontology files contain the GO term, GO ID, and the full formal definition (as well as other information like the reference PMID if there is one).  See http://geneontology.org/docs/download-ontology for more info on the various ontology files. Here's an example from go-basic.obo, which doesn't have as many relations as the full go.obo nor all the cross references in the go-plus.owl:

> [Term]
> id: GO:0000001
> name: mitochondrion inheritance
> namespace: biological_process
> def: "The distribution of mitochondria, including the mitochondrial genome, into daughter cells after mitosis or meiosis, mediated by interactions between mitochondria and the cytoskeleton." [GOC:mcc, PMID:10873824, PMID:11389764]
> synonym: "mitochondrial inheritance" EXACT []
> is_a: GO:0048308 ! organelle inheritance
> is_a: GO:0048311 ! mitochondrion distribution

You might find go-plus.owl (http://current.geneontology.org/ontology/extensions/go-plus.owl) to be the most informative for this, although it's not available in obo format shown above and can be a bit more difficult to parse/read.

**Reply by jasperhyp** [reply]
Thanks for answering!  Yes, I've looked into `go-basics.obo`.  The descriptions are good but usually short, and only 20% of the GO terms have PMID as sources (others are mostly defined by GOC).  I think `go-plus.owl` might have more xrefs, but that might point to too many different places and would be a nightmare for data curation haha.  Would `go-plus.owl` contain more direct/standard information like longer description, PMID, etc.?
